src/App.vue

```diff
@@ -517,12 +516,56 @@
         </div>
       </div>
     </div>
+    <div class="py-5">
+      <i-calendar
+        v-model="computedStuffingDate"
+        :disabled-previous-month="false"
+        :disabled-next-month="false"
```

Comment:
remove this props, disablePreviousMonth and disabledNextMonth should be not a prop (check i-datepicker)

Reply:
ok